lib/guardian/db/token.ex

```diff
@@ -7,9 +7,9 @@ defmodule GuardianDb.Token do
   import Ecto.Changeset
   import Ecto.Query, only: [from: 2]
 
-  alias GuardianDb.Token
+  alias Guardian.Db.Token
 
-  config = Application.get_env(:guardian_db, GuardianDb, [])
+  config = Application.get_env(:guardian_db, Guardian.Db, [])
```

Comment:
@doomspork wouldn't be better to read it under `guardian` key instead of `guardian_db`?

Basically `Guadian.DB` is the one that creates the scoping

Reply:
I'm not sure I follow @yordis, could you elaborate?

Reply:
Right now we are reading from `:guardian_db` `Application.get_env(:guardian_db, ...)` but I suggest to read from `Application.get_env(:guardian, ...)` and we use the module name for scoping the config anyway

But thinking on that right now, I am not sure if this is a good idea/good practice because this package name is `guardian_db` but it feels more natural to me to add it into `guardian` because we are using `Guardian.DB` for scoping the config instead of reading directly from `guardian_db` only

Reply:
Leave it as it is, I think this is a bad practice actually!

Reply:
@doomspork look this https://github.com/ueberauth/ueberauth_google/blob/master/lib/ueberauth/strategy/google/oauth.ex#L28 

even when the module is in another package we read it from `: ueberauth` key